solr/modules/ltr/src/test/org/apache/solr/ltr/store/rest/TestModelManager.java

```diff
@@ -202,4 +202,58 @@ public void testEndpointsFromFile() throws Exception {
     restTestHarness.delete(
         ManagedFeatureStore.REST_END_POINT + "/" + FeatureStore.DEFAULT_FEATURE_STORE_NAME);
   }
+
```

Comment:
in general it's not a good practice to have a test where you assert various scenarios and expectations, it's better to have 6 small tests than 1 big test.
Just think about debugging, if you get the big test to fail, and potentially the 5/6 assertion is broken, you need to debug trhough stuff you don't care.

If you have the tests separate, you can just dive in the broken one.

This is a best practice rarely followed in Solr, that is full of this example, but given we are contributing new code we should aim for the best :)